trajopt_sco/include/trajopt_sco/optimizers.hpp

```diff
@@ -101,7 +102,8 @@ struct BasicTrustRegionSQPParameters
   double max_merit_coeff_increases;
 
   double merit_coeff_increase_ratio;  // ratio that we increate coeff each time
-  double max_time;                    // not yet implemented
+  /** @brief Max time in seconds that the optimizer will run*/
+  double max_time;
```

Comment:
Should this be initialized to `std::numeric_limits<double>::max()`?

Reply:
Odd. Yeah it should be.

Reply:
Oh that is already being set in `BasicTrustRegionSQPParameters::BasicTrustRegionSQPParameters()`. That's why I didn't do it in this commit.   `max_time = static_cast<double>(INFINITY);`

Reply:
Ah ok.